news/2023/2023-08-28-medal-hunter-titles-second-coming.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "\"Medal Hunter\" Title's Second Coming"
+date: 2023-08-?? 03:00:00 +0000
+---

```

Comment:
```suggestion
---
```

Reply:
You can drag a range to suggest a multi-line fix, rather than what you've tried to do here.

Reply:
Thanks, noted. Not sure whats happening with the line breaks, remark is going crazy. I'll ask Wala/TicClick tonight

Reply:
;v